backend/Testing/Fixtures/IntegrationFixture.cs

```diff
@@ -13,11 +11,15 @@ namespace Testing.Fixtures;
 
 public class IntegrationFixture : IAsyncLifetime
 {
-    private static readonly string _templateRepoName = "test-template-repo.zip";
-    public FileInfo TemplateRepoZip { get; } = new(_templateRepoName);
-    public DirectoryInfo TemplateRepo { get; } = new(Path.Join(BasePath, "_template-repo_"));
+    private const string TemplateRepoZipName = "test-template-repo.zip";
+    public static readonly FileInfo TemplateRepoZip = new(TemplateRepoZipName);
+    public static readonly DirectoryInfo TemplateRepo = new(Path.Join(BasePath, "_template-repo_"));
     public ApiTestBase AdminApiTester { get; private set; } = new();
-    private string AdminJwt = string.Empty;
+
+    static IntegrationFixture()
```

Comment:
I see we're doing the delete here now. I think rather than using a bunch of static stuff it would make sense to setup a separate fixture and configure it as an [assembly fixture](https://xunit.net/docs/shared-context#assembly-fixture), then we could test it quite easily since we're not using a static constructor. We should be able to use dependency injection to get that fixture in this one in order to access it's properties.

Reply:
Hmmm...that's a v3 features, which is still [in alpha](https://xunit.net/docs/v3-alpha).
![image](https://github.com/sillsdev/languageforge-lexbox/assets/12587509/4a7e51be-666d-4bc3-871e-a8c2d26044dc)

Is it worth it?

Reply:
Oh you're right, ok let's get this merged.